HACKING.md

```diff
@@ -66,7 +66,7 @@ nickel-lang-lsp 0.1.0
 
 To only build the main crate `nickel-lang-core`, run:
 
-```console
+```shell
```

Comment:
Any reason for this change? It's not very important, but usually we've used `console` throughout the documentation because IIRC it's more adapted to showing multiple lines when some of them are output and some are commands (an interactive session, basically). In this case I'm not sure it makes any difference, but just for the sake of consistency

Reply:
It's just that this is the only code block in Hacking.md that uses `console` (and `shell` everywhere else) and I changed it for consistency. Perhaps I could instead change all `shell` to `console`?

Reply:
Oh, I see. Maybe I've been confusing it with README, and HACKING is older. It's fine then, consistency is better - we can always change everything at once in a later PR.